<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SkillInfo</class>
 <widget class="QWidget" name="SkillInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>199</width>
    <height>53</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="hLayoutMain">
   <item>
    <layout class="QVBoxLayout" name="vLayoutSkill1">
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vLayoutSkill2">
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vLayoutTripod">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="hLayoutTripodTitle">
       <property name="spacing">
        <number>1</number>
       </property>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hLayoutTripod">
       <property name="spacing">
        <number>1</number>
       </property>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="vLayoutRune">
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
